plugins/pulp_rpm/plugins/importers/yum/sync.py

```diff
@@ -632,6 +632,19 @@ def add_rpm_unit(self, metadata_files, unit):
             unit.save()
         except NotUniqueError:
             unit = unit.__class__.objects.filter(**unit.unit_key).first()
+
+        return unit
+
+    def filter_rpm_unit(self, unit):
```

Comment:
in my opinion this name of the function is somehow misleading, maybe 'pass_signature_filter' or similar?